src/elemeno_ai_sdk/ml/autofeatures/client.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict, Optional
+
+import aiohttp
+
+from elemeno_ai_sdk.ml.mlhub_client import MLHubRemote
+from elemeno_ai_sdk.utils import mlhub_auth
+
+
+class AutoFeaturesClient(MLHubRemote):
+    auto_features_ref = "YWktZmVhdHVyZS1lbmdpbmVlcmluZw=="
+    
+    def __init__(self, env: Optional[str] = None) -> None:
+        super().__init__(env=env)
+
+    async def get_job(self, job_id: str):
+        url = f"{self.base_url}/script-runner/{job_id}"
+        response = await self.get(url=url)
+        return response
+
+    async def list_jobs(self):
+        url = f"{self.base_url}/script-runner?ref={self.auto_features_ref}"
+        return await self.get(url=url)
+
+    async def run_job(self, filepath: str):
+        url = f"{self.base_url}/script-runner/{self.auto_features_ref}"
+
+        data = aiohttp.FormData()
+        file = open(filepath, "rb")
```

Comment:
here you can use the `with` clause to avoid the necessity to close the file in the end of the function

Reply:
The issue with using the `with` context here is because of the async nature of the MLHubClient requests. I would need to use a `async with`, which adds a bit of overhead in my opinion